packages/vaadin-overlay/test/focus-trap.test.js

```diff
@@ -105,18 +106,6 @@ describe('focus-trap', function () {
       expect(focusableElements[9]).to.eql(overlay.content.querySelector('vaadin-button'));
     });
 
-    it('should not focus next focusable element inside the content when Tab is prevented', async () => {
```

Comment:
It was previously possible to prevent the focus trap logic on `keydown` with `preventDefault()` called on the `keydown` event (see [here](https://github.com/vaadin/web-components/pull/3140/files#diff-a22b589817f0a0a072a03603aacbf01899a28200af52529f4d22879f213a5776L503)). That was meant to fix https://github.com/vaadin/vaadin-rich-text-editor/issues/125. However, it appears that the issue was fixed directly in the rich-text-editor editor at some point as the issue is not reproducible now, even without the workaround, so I decided that it can be dropped.